pkg/build/strategies/sti/sti.go

```diff
@@ -108,37 +108,37 @@ func New(req *api.Request) (*STI, error) {
 // An error represents a failure performing the build rather than a failure
 // of the build itself.  Callers should check the Success field of the result
 // to determine whether a build succeeded or not.
-func (b *STI) Build(request *api.Request) (*api.Result, error) {
-	defer b.garbage.Cleanup(request)
+func (b *STI) Build(config *api.Config) (*api.Result, error) {
+	defer b.garbage.Cleanup(config)
 
-	glog.V(1).Infof("Building %s", request.Tag)
-	if err := b.preparer.Prepare(request); err != nil {
+	glog.V(1).Infof("Building %s", config.Tag)
+	if err := b.preparer.Prepare(config); err != nil {
 		return nil, err
 	}
 
-	if b.incremental = b.artifacts.Exists(request); b.incremental {
-		glog.V(1).Infof("Existing image for tag %s detected for incremental build", request.Tag)
+	if b.incremental = b.artifacts.Exists(config); b.incremental {
+		glog.V(1).Infof("Existing image for tag %s detected for incremental build", config.Tag)
 	} else {
 		glog.V(1).Infof("Clean build will be performed")
 	}
 
-	glog.V(2).Infof("Performing source build from %s", request.Source)
+	glog.V(2).Infof("Performing source build from %s", config.Source)
 	if b.incremental {
-		if err := b.artifacts.Save(request); err != nil {
+		if err := b.artifacts.Save(config); err != nil {
 			glog.Warningf("Error saving previous build artifacts: %v", err)
```

Comment:
If I include the information about clean build in the first sentence it might get lost along the error. It's intentionally separate line to *mark* we will do a clean build for the user.